src/ansys/mapdl/core/cli/hpc.py

```diff
@@ -72,11 +73,26 @@
     help="""Python version to use to create the virtual environment and
 run the Python file. Python3 is used by default in the cluster.""",
 )
+@click.option(
+    "--inputs",
+    default=None,
+    type=str,
+    help="""
+    Input arguments for the simulation. You can specify several arguments by joining them with commas, therefore, strings defined this way cannot contain commas.
+    PyMAPDL will try to convert, whenever possible, these inputs to ints or floats, otherwise, they remind as strings. These arguments can be changed in the HPS website. For example: --inputs="force=123,value='mystring'"
+    """,
+)
+@click.option(
+    "--outputs",
+    default=None,
+    type=str,
+    help="""Output parameters. You can specify several arguments by joining them with commas. For example: --outputs="displacements,nodes".""",
+)
 @click.option(
     "--output_files",
     default=None,
     type=str,
-    help="""Output files to monitor.""",
+    help="""Output files to monitor. They can be specified as comma separated file names (so their names cannot contain commas). For example: --output_files="results.out,data.xls". """,
```

Comment:
```suggestion
    help="""Output files to monitor. Because you use commas to separate the file names, the names cannot contain commas. For example, ``--output_files="results.out,data.xls"``.""",
```